staging/src/k8s.io/pod-security-admission/policy/check_procMount_test.go

```diff
@@ -47,15 +49,36 @@ func TestProcMount(t *testing.T) {
 				HostUsers: &hostUsers,
 			}},
 			expectReason: `procMount`,
+			allowed:      false,
 			expectDetail: `containers "d", "e" must not set securityContext.procMount to "Unmasked", "other"`,
 		},
+		{
+			name: "procMount",
+			pod: &corev1.Pod{Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{Name: "a", SecurityContext: nil},
+					{Name: "b", SecurityContext: &corev1.SecurityContext{}},
+					{Name: "c", SecurityContext: &corev1.SecurityContext{ProcMount: &defaultValue}},
+					{Name: "d", SecurityContext: &corev1.SecurityContext{ProcMount: &unmaskedValue}},
+					{Name: "e", SecurityContext: &corev1.SecurityContext{ProcMount: &otherValue}},
+				},
+				HostUsers: &hostUsers,
+			}},
+			expectReason:                             "",
+			expectDetail:                             "",
+			allowed:                                  true,
+			enableUserNamespacesPodSecurityStandards: true,
+		},
 	}
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
+			if tc.enableUserNamespacesPodSecurityStandards {
+				RelaxPolicyForUserNamespacePods(true)
```

Comment:
set up a t.Cleanup to put this back when the test case is done

Reply:
done, as well as did so in other unit tests (which I based handling of this on)